install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -34,6 +67,65 @@ High-availability of storage in the infrastructure is left to the underlying
 storage provider.
 ====
 
+[[gfs-support-requirements]]
+== Support Requirements
+
+The following requirements must be met to create a supported integration of Red
+Hat Gluster Storage and {product-title}.
+
+[[gfs-supported-operating-systems]]
+=== Supported Operating Systems
+
+The following table lists the supported versions of {product-title} with Red Hat
+Gluster Storage Server.
+
+[options="header"]
+|===
+|Red Hat Gluster Storage | {product-title}
+
+|3.1.3
+|3.1 or later
+|===
+
+[[gfs-environment-requirements]]
+=== Environment Requirements
+
+The environment requirements for {product-title} and Red Hat Gluster Storage are
+described in this section.
+
+.Red Hat Gluster Storage
+- All installations of Red Hat Gluster Storage must have valid subscriptions to
+Red Hat Network channels and Subscription Management repositories.
+- Red Hat Gluster Storage installations must adhere to the requirements laid out
+in the
+link:https://access.redhat.com/documentation/en-US/Red_Hat_Storage/3.1/html/Installation_Guide/chap-Planning_Red_Hat_Storage_Installation.html[Red
+Hat Gluster Storage Installation Guide].
+- Red Hat Gluster Storage installations must be completely up to date with the
+latest patches and upgrades. Refer to the
+link:https://access.redhat.com/documentation/en-US/Red_Hat_Storage/3.1/html/Installation_Guide/index.html[Red
+Hat Gluster Storage 3.1 Installation Guide] to upgrade to the latest version.
+- The versions of {product-title} and Red Hat Gluster Storage integrated must be
+compatible, according to the information in
+xref:gfs-supported-operating-systems[Supported Operating Systems].
+- A fully-qualified domain name (FQDN) must be set for each hypervisor and Red Hat
+Gluster Storage server node. Ensure that correct DNS records exist, and that the
+FQDN is resolvable via both forward and reverse DNS lookup.
+
+.Red Hat OpenShift Enterprise
+- All installations of {product-title} must have valid subscriptions to Red Hat
+Network channels and Subscription Management repositories.
+- {product-title} installations must adhere to the requirements laid out in the
+xref:../../install_config/install/index.adoc#install-config-install-index[Installation
+and Configuration] documentation.
+- The {product-title} cluster must be up and running.
+- A user with *cluster-admin* permissions must be created.
+- All {product-title} nodes on RHEL systems must have the *glusterfs-fuse* RPM
+installed, which should match the version of Red Hat Gluster Storage server
+running in the containers. For more information on installing *glusterfs-fuse*,
```

Comment:
In the original version of this content, it listed the "glusterfs-client" RPM here, but I don't think that's right... IIRC we had that previously and changed our mention below to be "glusterfs-fuse", so I've done that here too. Also the RHGS doc link that follows only mentions "glusterfs-fuse" and not "glusterfs-client".